### PR TITLE
[CI:DOCS] install.md: include distro package info links

### PR DESCRIPTION
**Pull request by lsm5**
Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

### CODE REVIEW

**Review by mtrmac — COMMENTED** [review]
Thanks!

ACK at a first glance. I’ll actually _read_ the links later :)

**Comment by lsm5** [comment]
> Thanks!
> 
> ACK at a first glance. I’ll actually _read_ the links later :)

the links point to all open bugs for the skopeo package for that particular distro. Let me know if it should be of some other form.

**Comment by mtrmac** [comment]
I’m sorry, I didn’t mean to suggest there was anything wrong with them, I was just about to leave the computer at that time.

The link destinations are perfectly fine.

File: install.md [hunk 1]
```diff
@@ -1,20 +1,36 @@
...
+installation steps for many distros along with their info and support links.
 
 ### Fedora
 
 ```sh
 sudo dnf -y install skopeo
 ```
 
-### RHEL/CentOS ≥ 8 and CentOS Stream
+Package Info: [Dist-Git](https://src.fedoraproject.org/rpms/skopeo) and
```
Comment (mtrmac) [inline install.md:13]: Consider maybe just
```suggestion
[Package Info](https://src.fedoraproject.org/rpms/skopeo) and
```
— the link is useful even for non-git users.

**Review by mtrmac — COMMENTED** [review]
LGTM either way.

Feel free to merge (the PR is currently marked as draft), or wait for a review by a native speaker, as you like.

**Comment by lsm5** [comment]
@mtrmac ack, thanks for the suggestions, I can update those. I also need to add links to some other distros.

@TomSweeneyRedHat PTAL too.

File: install.md [hunk 2]
```diff
@@ -1,20 +1,36 @@
 # Installing Skopeo
 
 ## Distribution Packages
-`skopeo` may already be packaged in your distribution.
+`skopeo` may already be packaged in your distribution. This document lists the
+installation steps for many distros along with their info and support links.
```
Comment (TomSweeneyRedHat) [inline install.md:5]:
```suggestion
installation steps for many distros, along with their information and support links.
```

File: install.md (hunk 1 above)
Comment (TomSweeneyRedHat) [reply install.md:13]: I generally prefer spelling out `information` over `info` in formal documentation like this.  That said, `info` is not incorrect.

File: install.md [hunk 3]
```diff
@@ -1,20 +1,36 @@
...
 
 ```sh
 sudo dnf -y install skopeo
 ```
 
-### RHEL/CentOS ≥ 8 and CentOS Stream
+Package Info: [Dist-Git](https://src.fedoraproject.org/rpms/skopeo) and
+[Bugzilla](https://bugzilla.redhat.com/buglist.cgi?bug_status=__open__&classification=Fedora&component=skopeo&product=Fedora)
+
+Fedora bugs can be reported on the Skopeo GitHub itself.
```
Comment (TomSweeneyRedHat) [inline install.md:16]:
```suggestion
Fedora bugs can be reported on the Skopeo GitHub [Issues](https://github.com/containers/skopeo/issues) page.
```

File: install.md [hunk 4]
```diff
@@ -1,20 +1,36 @@
...
+
+### RHEL / CentOS Stream ≥ 8
 
 ```sh
 sudo dnf -y install skopeo
 ```
 
+If you're a RHEL customer, please reach out through the official RHEL support
```
Comment (TomSweeneyRedHat) [inline install.md:24]:
```suggestion
If you are a RHEL customer, please reach out through the official RHEL support
```

**Comment by lsm5** [comment]
@TomSweeneyRedHat done

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Pull request merged**